ios/FluentUI/Vnext/Divider/DividerTokens.swift

```diff
@@ -12,35 +12,41 @@ import UIKit
     case medium
 }
 
-/// Representation of design tokens to dividers at runtime which interfaces with the Design Token System auto-generated code
-/// Updating these properties causes the SwiftUI divider to updates its view automatically
-class MSFDividerTokens: MSFTokensBase, ObservableObject {
-    @Published public var color: UIColor!
-    @Published public var padding: CGFloat!
-
-    var spacing: MSFDividerSpacing
-
-    init(spacing: MSFDividerSpacing) {
+/// Design token set for the `FluentDivider` control.
+public class DividerTokens: ControlTokens {
+    /// Creates an instance of `MSFDividerTokens`with optional token value overrides.
+    /// - Parameters:
+    ///   - spacing: MSFDividerSpacing enumeration value that will define pre-defined value for the padding
+    ///   - padding: CGFloat that defines the padding around the Fluent Divider
+    ///   - color: The color of the Fluent Divider
+    public init(spacing: MSFDividerSpacing = .none,
```

Comment:
Seems like most of our other controls don't have a default value like this in the init, but I figured it was fine in this case.

Reply:
My concern is that these token files may eventually be generated from JSON, which will make these unique elements tricky.

And once the above issue is resolved (wherein `spacing` is always passed in from `defaultTokens`), there should be no need for a default value.